.github/workflows/main.yml

```
@@ -1,70 +1,64 @@
-          curl -o $HOME/.local/bin/ecs-deploy https://raw.githubusercontent.com/silinternational/ecs-deploy/master/ecs-deploy
-          chmod +x $HOME/.local/bin/ecs-deploy
+      - name: Setup node
+        uses: actions/setup-node@v5
+        with:
+          node-version: 24
 
-      - name: Tests
-        run: |
-          make test
+      - name: Build Docker image
+        uses: docker/build-push-action@v4
```

actionlint (1.7.11)

[error] the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)